clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -37,8 +37,18 @@
      * @param conf          The configuration.
      * @return              The new KafkaAdminClient.
      */
+    public static AdminClient create(AdminClientConfig conf) {
```

Comment:
Hmm, I just looked and I see that I forgot to include that change in that patch.

Thinking about it more, though, `ProducerConfig` and `ConsumerConfig` don't have public constructors, and it's not clear that `AdminClientConfig` should either.  it might be best to take a Java `Properties` object here.